query/reverse.js

```diff
@@ -11,9 +11,9 @@ function generate( params ){
   };
 
   var query  =  queries.distance( centroid, { size: params.size || 1 } );
-  query.sort = query.sort.concat(sort);
+  query.sort = query.sort.concat( sort() );
```

Comment:
why not pass ```params``` here? I know reverse doesnt have ```input``` field requirement. However, for consistency sake - I'd argue we pass ```params``` because maybe a script in the future might use ```params.lat``` ```params.lon``` for example..